test/unit/plotly-dash-preview_test.js

```diff
@@ -2,9 +2,14 @@ const tap = require('tap')
 const sinon = require('sinon')
 
 const _module = require('../../src/component/plotly-dash-preview')
+const constants = require('../../src/component/plotly-dash-preview/constants')
 const remote = require('../../src/util/remote')
 const { createMockWindow } = require('../common')
 
+function clone (obj) {
+  return JSON.parse(JSON.stringify(obj))
+}
```

Comment:
I know this is only test code, but this `clone` function is dirty (and slow). 

It might not be worth adding another dev dep to use an "extend deep" helper, but here looks like you can simply turn `const mock` the object into a function returning a new mock object:

```js
const mock = () => ({
  url: 'https://dash-app.com',
  selector: 'dummy'
})
```

Reply:
Done in 156a35cc5e6111a85910fa47fa6eb670db5708f8